src/screens/LinkContextScreen/elements/Footer.js

```diff
@@ -157,7 +157,7 @@ const styles = StyleSheet.create({
   wrapperFooter: {
     paddingHorizontal: 8,
     height: 52,
-    borderBottomColor: COLORS.gray1,
+    borderBottomColor: COLORS.balance_gray,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.gray1` to `COLORS.balance_gray` seems fine as long as the new color `balance_gray` is defined in the `COLORS` object and it fits the design requirements. Please ensure that this color change is consistent with the overall theme of the application.
```diff
-    borderBottomColor: COLORS.gray1,
+    borderBottomColor: COLORS.balance_gray,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->